packages/opentelemetry-propagator-jaeger/test/JaegerPropagator.test.ts

```diff
@@ -159,7 +159,7 @@ describe('JaegerPropagator', () => {
       });
     });
 
-    it('should extract context of a sampled span from carrier with 1 bit flag', () => {
+    it('should extract context of a sampled span from carrier with 1 bit flag(1)', () => {
```

Comment:
nit: What does it mean "1 bit flag"? Should it say "1 hex digit flag" or "1 character flag"?

Reply:
Yes, the original test description string is a bit ambiguous.
I think it would be more accurate to use "a single digit character" to describe this, right?

Reply:
I think it is "with flag 1 set" and `1` is just the example. Could easily have been `3`

Reply:
Is there anything else I can do?
There are very few flag bits that can cause problems in production

Reply:
```suggestion
    it('should extract context of a sampled span from carrier with sampled bit (bit 1) set', () => {
```

Reply:
Actually, I think the essence of the test is to check that the flags work when encoded with a single hex character (the thing this PR fixes).
The previous test ("should extract context of a sampled span from carrier") covers the case where flags are 2 digits.

I am good with any name, it's really a nit, don't want to block anything.
If you change the name, please do it also for the other (new) test as well.

Reply:
Agree
This is just a description issue, and any developer who sees this test case will not misunderstand it.
We need to solve the problem that the traceFlag length can be 1 and 2 because of the standard definition and compatibility.
In fact, there are four possible values, 0/00/01/1. If some value such as 02 or 2 appear, it is not the library's problem, but the developer's problem who generated the trace header

Reply:
Not sure I understand.
The PR is great, I only suggested improving the test name as it is currently not very clear.
If you want to do it - amazing. If not, let's merge it as is :)

Reply:
let's merge it as is :)